packages/api/src/index.ts

```diff
@@ -18,6 +18,14 @@ app.use(express.json());
 
 const GitHubStrategy = require('passport-github2');
 
+const contributionPollTimer = (milliseconds: number) => {
+  logger.info(`Starting new contribution poll timer: ${milliseconds}ms`);
+  setTimeout(async () => {
+    await fetch(`${env.appUrl}/api/contributions`);
```

Comment:
We definitely shouldn't be making calls to the api from the api 😅 Check the general review comments for suggestions

Reply:
done